packages/office-addin-sso/src/configure.ts

```diff
@@ -17,11 +15,9 @@ export async function createNewApplication(ssoAppName: string, port: string, use
         const rePort = new RegExp('{PORT}', 'g');
         azRestCommand = azRestCommand.replace(reName, ssoAppName).replace(rePort, port);
         const applicationJson: Object = await promiseExecuteCommand(azRestCommand, true /* returnJson */);
-        usageDataHelper.sendUsageDataSuccessEvent('createNewApplication');
         return applicationJson;
     } catch (err) {
         const errorMessage: string = `Unable to register new application: \n${err}`
-        usageDataHelper.sendUsageDataException("createNewApplication", errorMessage);
```

Comment:
I think we may still want the exception telemetry here so we get a granular sense of where the error occurs